website/pages/en/querying/querying-the-graph.mdx

```diff
@@ -12,7 +12,27 @@ Each subgraph published to The Graph Network has a unique query URL in Graph Exp
 
 ![Query Subgraph Pane](/img/query-subgraph-pane.png)
 
+<<<<<<< HEAD
```

Comment:
@itsjerryokolo looks like you need to fix the conflict here?

Reply:
Let me fix that